packages/@aws-cdk/aws-synthetics/test/canaries/python/canary.py

```diff
@@ -0,0 +1,59 @@
+import json
```

Comment:
is this file sourced from somewhere? or did you handwrite it? if the former, we'll have to look in to how to properly attribute